plenum/bls/bls_bft_replica_plenum.py

```diff
@@ -198,6 +266,29 @@ def _sign_state(self, pre_prepare: PrePrepare):
     def _can_calculate_multi_sig(self,
                                  key_3PC,
                                  quorums) -> bool:
+        if key_3PC in self._all_signatures:
+            sigs_for_request = self._all_signatures[key_3PC]
+            sigs_invalid = list(
+                filter(
+                    lambda item: not quorums.bls_signatures.is_reached(len(list(item[1].values()))),
+                    sigs_for_request.items()
+                )
+            )
+            if sigs_invalid:
```

Comment:
What if we have quorum for some ledgers, not for all? Currently `False` is returned, but maybe we should be able to calculate multi-sig for the sigs we have quorum for?